cmd/river/go.mod

```diff
@@ -23,10 +24,9 @@ require (
 	github.com/jackc/pgpassfile v1.0.0 // indirect
 	github.com/jackc/pgservicefile v0.0.0-20221227161230-091c0ba34f0a // indirect
 	github.com/jackc/puddle/v2 v2.2.1 // indirect
-	github.com/oklog/ulid/v2 v2.1.0 // indirect
+	github.com/lmittmann/tint v1.0.4 // indirect
```

Comment:
So ... I brought in this package for use with the benchmarker, which is a colorized log handler for slog. I was debugging some benchmark problems and this makes things _far_ easier to read because the default Go output is such absolute garbage.

The nice thing about Tint is that it itself has zero dependencies, and we only need it for the River CLI package without having to add it to the top level one, so I think it's okay.

Here's a screenshot on the benchmarker running in verbose (I also added a `--debug` that makes it even more verbose):

<img width="929" alt="image" src="https://github.com/riverqueue/river/assets/96890/b4a6a5d9-d310-4e75-a631-2aa345e5c446">

Reply:
Looks solid! I think I'm gonna pull this into my riverdemo app too.